Strongbox/Classes/Strongbox.swift

```diff
@@ -155,3 +155,52 @@ public class Strongbox {
     }
 
 }
+
+// Codable
+
+extension Strongbox {
+    public func encode<T: Encodable>(_ encodable: T?, key: String, encoder: JSONEncoder = .init(), accessibility: CFString = kSecAttrAccessibleWhenUnlocked) throws {
+        let sb = Strongbox()
+        guard
+            let encodable = encodable
+            else { sb.archive(nil, key: key, accessibility: accessibility) ; return }
+        let data = try encoder.encode(encodable)
+        sb.archive(data, key: key, accessibility: accessibility)
+    }
+    
+    public func decode<T: Decodable>(forKey key: String, decoder: JSONDecoder = .init()) throws -> T? {
+        let sb = Strongbox()
+        guard
+            let data = sb.unarchive(objectForKey: key) as? Data
+            else { return nil }
+        return try decoder.decode(T.self, from: data)
+    }
+}
+
+// Convenience 
+
+extension Strongbox {
```

Comment:
Move this stuff into the main definition, and move the Codable extension below it into the first Codable extension. That seems logically better to me.